adminSiteServer/migrateMultiDimConfigs.ts

```diff
@@ -0,0 +1,29 @@
+import {
+    DbPlainMultiDimDataPage,
+    MultiDimDataPagesTableName,
+} from "@ourworldindata/types"
+import { knexReadWriteTransaction, TransactionCloseMode } from "../db/db.js"
+import { createMultiDimConfig } from "./multiDim.js"
+
+/**
+ * Migrates the old multi-dim config to a normalized format, creates related
+ * records in `chart_configs` and the `multi_dim_x_chart_configs` tables and
+ * uploads the full chart config of each view and the multi-dim config to R2.
+ *
+ * The old config must be valid, otherwise the migration will fail.
+ *
+ * This isn't an actual DB data migration because of circular dependencies
+ * between `adminSiteServer/multiDim` and `db`.
+ */
+async function main() {
```

Comment:
Will move to `devTools`.

It could be a DB data migration (as written), not a DB schema migration. Could you say more about what would you change about the wording?